CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ master
 ------
 
 * Drop support for Ruby `< 2.1.0` and Rails `4.0.0, < 3.2.0`
+* Introduce `rails g ember-cli:heroku` generator to setup a project for
```

Comment:
I made this comment and then Github had some saving problems.

How about "to set up a project for Heroku deployment"?  ("setup" is a noun, "set up" is a verb)